contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -128,7 +128,13 @@ contract RaidenMicroTransferChannels {
         return str;
     }
 
-    // 56014 gas cost
+    /*
+     *  Temporary implementation of verifyBalanceProof.
+     *  Message string reproduction is a workaround
+     *  until https://github.com/ethereum/EIPs/pull/712 is implemented.
```

Comment:
Just a note: That EIP hasn't been accepted yet, so it's not certain that it will ever be implemented.